crates/orchestrator/src/lib.rs

```diff
@@ -334,7 +334,11 @@ where
     fn post_ready(&mut self) -> Result<(), ServerError> {
         self.nodes_connected += 1;
         println!("Nodes connected: {}", self.nodes_connected);
-        if self.nodes_connected >= (self.config.config.num_nodes_with_stake.get() as u64) {
+        // i.e. nodes_connected >= num_nodes_with_stake * (start_threshold.0 / start_threshold.1)
+        if self.nodes_connected * self.config.config.start_threshold.1
```

Comment:
How come we aren't just doing the formula as described in the comment?

Reply:
You can't divide a `u64` and compare to a `u64`, and I think casting to `f64` is best avoided. You also don't have to worry about 0 in the denominator this way.